packages/datadog-instrumentations/src/next.js

```diff
@@ -214,7 +214,7 @@ addHook({
 
 addHook({
   name: 'next',
-  versions: ['>=11.1'],
+  versions: ['>=11.1 <12.3.5', '>=13'],
```

Comment:
This will mean that we have a gap of support in the next release.

Reply:
yes as discussed i don't see a way to have the CI pass without doing that. This is just a temporary CI fix and i'm monitoring the issue on nextjs repo to revert that asap

Reply:
you said it yourself, if the tip of the range isn't available, it doesn't work. And if the test ranges are different from the instrumentation ranges, the CI checker complains too

Reply:
We should never drop a version from the instrumentation unless it's completely broken (meaning the instrumented library itself cannot work to begin with), but not if it's fixable, and my understanding is that the broken Next release is fixable by release the missing dependencies, in which case a version would be out that our next release doesn't support.

> yes as discussed i don't see a way to have the CI pass without doing that. This is just a temporary CI fix and i'm monitoring the issue on nextjs repo to revert that asap

A temporary fix should not make the main branch unreleasable, which this would do.

Reply:
I understand, this isn't blocking me, you can close the PR and find a better fix if you want